content/en/docs/overview/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Overview"
+weight: 200
+---
+
+In this chapter, we're going to get an overview which covers the most important language features of Go.
+This is only an overview and by no means a complete list.
+Since it is hard to learn a lot of new language features at once, we deliberatly left out certain constructs which, based on our expirience, are not used very often and which are not required most of the time.
+But even with this limited selection of features you should be able to understand most of the Go programs and solve a wide range of programming problems.
+
+If you are intressted in more details when going over the topics or when you come across a syntax construct somewhere else which we did not cover you can always reach out to the official Go resources which cover every feature in great detail:
+
+* The [Tour of Go](https://go.dev/tour/list) is similar to the overview in this chapter but it covers more details. It goes over most of the language features by looking at small examples.
```

Comment:
I would probably leave this out here and maybe add a page that includes all external links at the end. We want people to stay on our course.

Reply:
I removed the second part which said that people can use the Tour of Go instead of our introduction but kept the two links to the resources. This way interested people know where they can find more detailed information.